### PR TITLE
Update feedaty.php

### PR DESCRIPTION
**Pull request by feedaty**
added support per SSL websites that include our plugin
updated version

### CODE REVIEW

File: feedaty.php [hunk 1]
```diff
@@ -594,7 +594,7 @@ public function fdGetProductData($id)
 	private function fdGenerateStars($data)
 	{
 		if (!isset($data)) $data = 0;
-		return '<img src="http://'.'www.feedaty.com/rating/rate-small-'.(int)$data.'.png" height="15">';
+		return '<img src="//'.'widget.zoorate.com/rating/rate-small-'.(int)$data.'.png" height="15">';
```
Comment (Quetzacoalt91) [inline feedaty.php]: The concat between `//` and `widget.zoorate` is not useful and should be removed.

**Comment by feedaty** [comment]
ok done thanks

**Comment by Quetzacoalt91** [comment]
Hi @feedaty,

Thanks, the PR has just been merged and the changes will be available tomorrow.
Next time, do not forget to update the config.xml ;)

Regards

**Comment by feedaty** [comment]
thank you,
sorry but we changed developer lately

**Pull request merged**